src/nl/hannahsten/texifyidea/psi/LatexCommandsImplUtil.kt

```diff
@@ -222,7 +222,9 @@ fun hasLabel(element: LatexCommands): Boolean {
 }
 
 fun setName(element: LatexCommands, newName: String): PsiElement {
-    val newText = element.text.replace(element.name ?: return element, newName)
+    var newText = element.text.replace(element.name ?: return element, newName)
+    if (!newText.startsWith("\\"))
+        newText = "\\" + newText
```

Comment:
Yes, this should be safe to do.

btw, completely unrelated but did you know that if you inline rename (like with labels) an element, if you press shift+F6 again the dialog will pop up? I didn't, but now I do. And then to think that's why `LatexRefactoringSupportProvider#isMemberInplaceRenameAvailable` does not accept LatexCommands.

Reply:
TIL :joy: